SU2_CFD/include/limiters/CLimiterDetails.hpp

```diff
@@ -87,6 +87,36 @@ struct LimiterHelpers
     Type factor = 0.5*(1.0+dist+sin(PI_NUMBER*dist)/PI_NUMBER);
     return max(0.0, min(factor, 1.0));
   }
+
+  FORCEINLINE static Type R3Function(const Type& proj, const Type& delta, const Type& epsp)
+  {
+    Type Dp = fabs(delta); Type Dm = fabs(proj);
+    if(Dp>(2.0*Dm)) {return 1.0;}
+    else {
+        Type y = Dp*Dp*Dp + epsp;
+        Type S3 = 4.0*Dm*Dm;
+        return ((y + Dp*S3) / (y + Dm*(delta*delta+S3)));}
+  }
+
+  FORCEINLINE static Type R4Function(const Type& proj, const Type& delta, const Type& epsp)
+  {
+    Type Dp = fabs(delta); Type Dm = fabs(proj);
+    if(Dp>(2.0*Dm)) {return 1.0;}
+    else {
+        Type y = Dp*Dp*Dp*Dp + epsp;
+        Type S4 = 2.0*Dm*(Dp*Dp-2.0*Dm*(Dp-2.0*Dm));
+        return ((y + Dp*S4) / (y + Dm*(delta*delta*delta+S4)));}
```

Comment:
+ use pow for the higher powers please
```suggestion
        Type y = pow(Dp,4) + epsp;
        Type S4 = 2.0*Dm*(Dp*Dp-2.0*Dm*(Dp-2.0*Dm));
        return ((y + Dp*S4) / (y + Dm*(pow(delta,3)+S4)));}
```